.gitignore

```diff
@@ -12,3 +12,6 @@ docs/man
 npm-debug.log
 .project
 test/memory.json
```

Comment:
Is this verified @Amir-61? If this is the case, maybe you can update globalization.md so we can all follow suit.